components/automate-ui/src/app/modules/team/team-details/team-details.component.spec.ts

```diff
@@ -112,14 +91,7 @@ describe('TeamDetailsComponent v2', () => {
       imports: [
         ReactiveFormsModule,
         RouterTestingModule,
-        StoreModule.forRoot({
-          router: routerReducer,
-          teams: teamEntityReducer,
-          users: userEntityReducer,
-          policies: policyEntityReducer,
-          projects: projectEntityReducer,
-          projectsFilter: projectsFilterReducer
-        }, { initialState, runtimeChecks })
+        StoreModule.forRoot(ngrxReducers, { initialState, runtimeChecks })
```

Comment:
By the time I fixed the two bugs that prevented further progress here (in the second PR I mentioned), I realized a cleaner way was to get rid of the itemized properties and just use the globally defined `defaultInitialState`, which already includes the two needed lines (for client runs and notifications). Here it is using a customized version of that global state (`initialState`).